modules/openapi-generator/src/main/resources/php-nextgen/api.mustache

```diff
@@ -185,7 +187,7 @@ use {{invokerPackage}}\ObjectSerializer;
     {{#vendorExtensions.x-group-parameters}}
         array $associative_array
     {{/vendorExtensions.x-group-parameters}}
-    ): {{{vendorExtensions.x-php-return-type}}}
+    ){{#returnType}}: {{{vendorExtensions.x-php-return-type}}}{{/returnType}}
```

Comment:
maybe we should keep it as seems like that's what the php community prefers: https://stackoverflow.com/a/33671413

Reply:
Please note we get this: return type without `return` statement: https://github.com/conedevelopment/billingo-php/blob/0504445fd3f060dedcb36ec0b7b1e04d12ca26c5/src/Api/BankAccountApi.php#L714-L721

Reply:
ah ok. looks like a different issue ...

i'll take a further look